.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
 
     // transpiled JavaScript
     "lib": true,
-  }
+  },
+  "editor.formatOnSave": true,
```

Comment:
Get Prettier to run on save.

Reply:
I was playing around with this at https://github.com/dsp-testing/qc-run2/pull/2

Don't you also need to set the default formatter? https://github.com/dsp-testing/qc-run2/commit/0bc72f40f1e5045e9d5083de75aa690cd18b2292

I was also unclear on how this would interact with eslint -- I think that _might_ be configured to run prettier as its own rules. And either way they might stomp on each other if they are both running on save.

Would you mind doing the prettier changes as a separate PR so I can try them out in isolation?